unit_tests/test_config.py

```diff
@@ -681,15 +681,15 @@ def test_21_nested_values(self):
                          {'alternator-dns': 'scylladb/hydra-loaders:alternator-dns-0.1',
                           'cassandra-stress': 'scylla-bench',
                           'cdc-stresser': 'scylladb/hydra-loaders:cdc-stresser-A',
-                          'gemini': 'scylladb/hydra-loaders:gemini-v1.8.4',
+                          'gemini': 'scylladb/hydra-loaders:gemini-v1.8.6',
```

Comment:
maybe we should change this test, to stop annoying us each release 
i.e. to overwrite all of the stress images, so it would be fixed, regardless of the defaults.